lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -836,53 +835,75 @@ Display simple command outputs from all DB hosts using B<ansible> command.
 
 =item * B<sap_sid>: SAP system ID. Default 'SAP_SID'
 
+=item * B<scenarios>: ARRAYREF with list of installed components
+
 =back
 =cut
 
-sub ansible_hanasr_show_status {
+sub ansible_show_status {
     my (%args) = @_;
     croak 'Missing mandatory argument "sdaf_config_root_dir".' unless $args{sdaf_config_root_dir};
```

Comment:
we have this nice pattern for multiple argument validation https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/d1bc97fa922e19610d855a3208ff6910fedb7168/lib/sles4sap/ipaddr2.pm#L226